### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by MDVAurelien**
Progression of the french translation.  I'll be back after a few months but I'm going to progress much faster in the next few days.

### CODE REVIEW

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
I notice this one is for freenas/11.2, I would suggest mainly focusing on master at this point, future releases will be cut from there

**Comment by MDVAurelien** [comment]
Ok so I'm focusing on this one: https://github.com/freenas/webui/blob/master/src/assets/i18n/en.po?

**Comment by erinix** [comment]
correct :)

**Comment by MDVAurelien** [comment]
it's gone ;-) Thanks for the information

File: src/assets/i18n/fr.po [hunk 1]
```diff
@@ -416,19 +452,29 @@ msgid ""
 "<i>global</i> is a reserved name that cannot be used as a share name. "
 "Please enter a different share name."
 msgstr ""
+"<i>global</i> est un nom réservé qui ne peut pas être utilisé comme nom de partage. "
+"Veuillez entrer un nom d'action différent."
```
[the remark this answers is not in the document]
Comment (MDVAurelien) [reply src/assets/i18n/fr.po]: Merci effectivement, je suis en train de reprendre depuis le début. Je suis preneur de demande de corrections parce que je vais traduire le tout le fichier assez rapidement.

File: src/assets/i18n/fr.po [hunk 2]
```diff
@@ -142,13 +142,22 @@ msgid ""
 "target=\"_blank\">Encryption section</a> of the guide before activating "
 "this option."
 msgstr ""
+"Le cryptage<a href=\"https://www.freebsd.org/cgi/man.cgi?query=geli&manpath=FreeBSD"
```
[the remark this answers is not in the document]
Comment (MDVAurelien) [reply src/assets/i18n/fr.po]: Oui absolument.

File: src/assets/i18n/fr.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (MDVAurelien) [reply src/assets/i18n/fr.po]: Si tu souhaites visualiser je vais traduire directement depuis la branche Master: https://github.com/freenas/webui/blob/master/src/assets/i18n/fr.po?

[the remark this answers is not in the document]
Comment (MDVAurelien) [reply src/assets/i18n/fr.po]: Je viens de proposer ce commit si tu veux y jeter un oeil : https://github.com/freenas/webui/pull/2799/files

[the remark this answers is not in the document]
Comment (rds13) [reply src/assets/i18n/fr.po]: Désolé, j'ai loupé mon retour sur ton travail mais j'ai apprécié ta proposition, merci.

**Comment by erinix** [comment]
fyi, there is now a freenas/11.3-stable branch too, any translations you want in 11.3 will need to go in there

**Comment by MDVAurelien** [comment]
Hi, the translation of the master branch will be reported on the branch freenas / 11.3-stable too? Or do I have to translate both now?

**Comment by erinix** [comment]
actually nevermind that last message, we have not branched yet, just keep using master

**Comment by MDVAurelien** [comment]
thanks boss ;-)

**Pull request merged**